### PR TITLE
Adding asgiref as mandatory package while installing python sqlcommen…

### PR DESCRIPTION
**Pull request by Thiyagu55**

### CODE REVIEW

File: python/sqlcommenter-python/setup.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def read_file(filename):
     long_description_content_type='text/markdown',
     license='BSD',
     packages=find_packages(exclude=['tests']),
+    install_requires=['asgiref'],
```
Comment (sjs994) [inline python/sqlcommenter-python/setup.py:37]: Do we need to update this file instead [python/sqlcommenter-python/tox.ini ?](https://github.com/google/sqlcommenter/blob/master/python/sqlcommenter-python/tox.ini)

Comment (Thiyagu55) [reply python/sqlcommenter-python/setup.py:37]: We either update setup.cfg or setup.py. But we didn't use setup.cfg here.  Refer https://github.com/open-telemetry/opentelemetry-python-contrib/blob/7bb29bd9ab81baf941a81619baaa77c0fa527f76/instrumentation/opentelemetry-instrumentation-urllib/setup.py#L28

Comment (Thiyagu55) [reply python/sqlcommenter-python/setup.py:37]: We shall create a new PR for setup.cfg migration

Comment (Thiyagu55) [reply python/sqlcommenter-python/setup.py:37]: Updated  also in tox.ini

**Review by sjs994 — APPROVED** [review]

**Pull request merged**